src/SFA.DAS.Commitments.Api/SFA.DAS.Commitments.Api.csproj

```diff
@@ -281,6 +295,9 @@
     <Content Include="Global.asax" />
     <Content Include="scripts\ai.0.15.0-build58334.js" />
     <Content Include="scripts\ai.0.15.0-build58334.min.js" />
+    <None Include="SFA.DAS.Commitments.Api.dll.config">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
Why do we need to copy this?

Reply:
The commitments api was failing to deploy, because of the System.Net.Http issue already mentioned.
We needed the binding redirect to fix the deployment, but the way cloud services work, the binding redirects in the web.config are ignored, and instead you need to add a new config file for them. More details in this blog post...
https://blogs.msdn.microsoft.com/friis/2014/05/15/webrole-entry-point-and-config-file/